feature/experimental/policy/policy_base/base_vrf_selection_test.go

```diff
@@ -0,0 +1,393 @@
+		s6.Enabled = ygot.Bool(true)
+	}
+	s6.GetOrCreateAddress(me.IPv6).PrefixLength = ygot.Uint8(plen6)
+
+	return i
+}
+
+// Configure Network instance on the DUT
+func configNetworkInstance(name string, peer *attrs.Attributes) *telemetry.NetworkInstance {
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(name)
+
+	ni.Type = telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2L3
+	static := ni.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC")
+	ipv4Nh := static.GetOrCreateStatic("0.0.0.0/0").GetOrCreateNextHop(peer.IPv4)
+	ipv4Nh.NextHop, _ = ipv4Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv4)
+	ipv6Nh := static.GetOrCreateStatic("::/0").GetOrCreateNextHop(peer.IPv6)
+	ipv6Nh.NextHop, _ = ipv6Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv6)
+	ipv4Nh.Recurse = ygot.Bool(true)
+	ipv6Nh.Recurse = ygot.Bool(true)
+
+	return ni
+}
+
+func configForwardingPolicy() *telemetry.NetworkInstance_PolicyForwarding {
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
+	ipv4Address := "0.0.0.0/0"
+	ipv6Address := "::/0"
+	ipv6dest := "2001:db8::1/128"
+
+	// Match policy
+	policyFwding := ni.GetOrCreatePolicyForwarding()
+
+	fwdPolicy1 := policyFwding.GetOrCreatePolicy("match-ipv4")
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6dest)
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateAction().Discard = ygot.Bool(false)
+	fwdPolicy1.GetOrCreateRule(3).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
+	fwdPolicy1.GetOrCreateRule(3).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
+	fwdPolicy1.GetOrCreateRule(3).GetOrCreateAction().Discard = ygot.Bool(true)
+
+	fwdPolicy2 := policyFwding.GetOrCreatePolicy("match-ipip")
+	fwdPolicy2.GetOrCreateRule(1).GetOrCreateIpv4().Protocol = telemetry.UnionUint8(ipipProtocol)
+	fwdPolicy2.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
+	fwdPolicy2.GetOrCreateRule(2).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
+	fwdPolicy2.GetOrCreateRule(2).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
+	fwdPolicy2.GetOrCreateRule(2).GetOrCreateAction().Discard = ygot.Bool(true)
+
+	fwdPolicy3 := policyFwding.GetOrCreatePolicy("match-ip4ip6")
+	fwdPolicy3.GetOrCreateRule(1).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
+	fwdPolicy3.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
+	fwdPolicy3.GetOrCreateRule(2).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
+	fwdPolicy3.GetOrCreateRule(2).GetOrCreateAction().NetworkInstance = ygot.String("20")
+	fwdPolicy3.GetOrCreateRule(3).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
```

Comment:
Rule 3 seems unreachable since rule 1 covers ipv4 and rule 2 covers ipv6.

Reply:
Done, in new commit. Thank You.